content/about/code-of-conduct.md

```diff
@@ -2,4 +2,105 @@
+> (CC-BY-NC 2.5 license) which was adapted from the example policy at the
+> [Geek Feminism wiki](https://geekfeminism.wikia.org/wiki/Conference_anti-harassment/Policy),
+> created by the Ada Initiative and other volunteers (CC-0 license).
+> The procedure for reporting harassment has been adopted from the Ada Initiative's guide titled
+> ["workshop anti-harassment/Responding to Reports"](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports).
+
+We value the participation of each stakeholder and want our members and
+participants to our events to have an enjoyable and fulfilling experience.
+Accordingly, all members and participants are expected to show respect and
+courtesy to other members and participants through all communication channels.
+
+To make clear what is expected, all Nordic-RSE members as well as participants,
+speakers, exhibitors, organisers and volunteers at Nordic-RSE events are
+required to conform to the following code of conduct. Organisers will enforce
+this code throughout the event.
+
+
+### Summary
+
+We are dedicated to providing a harassment-free experience for everyone. We do
+not tolerate harassment in any form.
+
+All communication should be appropriate for a professional audience including
+people of many different backgrounds.
+
+Be kind to others. Do not insult or put down others.
+
+Behave professionally. Remember that harassment, unprofessional remarks and
+messages, and exclusionary jokes are not appropriate in Nordic-RSE.
+
+Members violating these rules may have their membership revoked by the board.
+Participants to events violating these rules may be asked to leave without a
+refund at the sole discretion of the conference organisers.
+
+Thank you for helping make this a welcoming, friendly event for all.
+
+
+### Clarifications
+
+Harassment includes offensive communication related to gender, sexual
+orientation, disability, physical appearance, body size, race, religion, sexual
+images in public spaces, deliberate intimidation, stalking, following,
+harassing photography or recording, sustained disruption of talks or other
+events, inappropriate physical contact, and unwelcome sexual attention.
+
+Participants asked to stop any harassing behaviour are expected to comply
+immediately.
+
+Be careful in the words that you choose. Remember that words can be offensive
+to those around you. Offensive jokes are not acceptable in the Nordic-RSE.
+Excessive swearing is not appropriate.
+
+If a member engages in behaviour that violates this code of conduct, the board
+of the association may take any action they deem appropriate, including
+warning the offender or may revoking their membership without refund of the
+membership fee.
+
+If a participant at a Nordic-RSE event violates this code of conduct,
+organisers may take any action they deem appropriate, including
+warning the offender or expulsion from the workshop with no refund.
```

Comment:
Should we also specifically cover the case where a board member is involved?
H/she is a member of the association so we are covered but this person should also be excluded from the board (and should not be involved in the exclusion decision).

Reply:
I added a chapter about this. What do you think?